cmake.deps/cmake/BuildLuajit.cmake

```diff
@@ -41,7 +41,16 @@ if(APPLE)
   set(DEPLOYMENT_TARGET "MACOSX_DEPLOYMENT_TARGET=${CMAKE_OSX_DEPLOYMENT_TARGET}")
 endif()
 
-if(UNIX)
+if(CYGWIN)
+  BuildLuajit(INSTALL_COMMAND ${BUILDCMD_UNIX}
+    CC=${DEPS_C_COMPILER} PREFIX=${DEPS_INSTALL_DIR}
+    ${DEPLOYMENT_TARGET} install
+    COMMAND ${CMAKE_COMMAND} -E copy ${DEPS_BUILD_DIR}/src/luajit/src/cyglua51.dll ${DEPS_BIN_DIR}
+    COMMAND ${CMAKE_COMMAND} -E copy ${DEPS_BUILD_DIR}/src/luajit/src/libluajit-5.1.dll.a ${DEPS_LIB_DIR}
+    # Luarocks searches for lua51.dll in lib
+    COMMAND ${CMAKE_COMMAND} -E copy ${DEPS_BUILD_DIR}/src/luajit/src/cyglua51.dll ${DEPS_LIB_DIR})
+
```

Comment:
I think this is missing the `CopyFilesGlob.cmake` commands the other windows builds have for the extra `share` files luajit ships with. I don't think you'll notice them missing unless you specifically try to call on a function from them like in `p.lua`.